include/arch/log.h

```diff
@@ -0,0 +1,11 @@
+/* log.h - automatically selects the correct log.h file to include */
```

Comment:
These doesn't seem to be a need for this file. You can simply include `arch/xtensa/log.h` directly as what you have is only for Xtensa and none of the other architectures.

Reply:
There really is not need for this file at this point, as what you have here is only for Xtensa (and not for multiple arch).

Reply:
Have removed this file as per suggestion